import funciones
"""5. Sabiendo que ValueError es la excepción que se lanza cuando no podemos convertir una 
cadena de texto en su valor numérico, escriba una función get_int() que lea un valor entero 
del usuario y lo devuelva, iterando mientras el valor no sea correcto. Intente resolver el 
ejercicio tanto de manera iterativa como recursiva."""

print("En este programa podes ingresar una cantidad de números hasta que ingreses una letra")
lista_enteros = []
funciones.get_int(lista_enteros)
#Funciona